pubspec.yaml

```
@@ -11,10 +11,15 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  bloc: ^6.1.0 # BLoC (Business Logic Component) for Flutter State Management
+  # BLoC (Business Logic Component) for Flutter State Management
+  bloc: ^6.1.0
```

🟤 
Move these to the right side. this reduces lines of code and makes debugging easier.
```suggestion
  bloc: ^6.1.0 # BLoC (Business Logic Component) for Flutter State Management
```

---

done